mods/doors/init.lua

```diff
@@ -651,10 +663,14 @@ function doors.register_trapdoor(name, def)
 		def.tile_side
 	}
 
-	def_opened.node_box = {
-		type = "fixed",
-		fixed = {-0.5, -0.5, 6/16, 0.5, 0.5, 0.5}
-	}
+	if def.opened and def.closed then
```

Comment:
```suggestion
	if def.opened then
```

Reply:
Same as previous suggestion.

Reply:
This is fine as is, both nodeboxes should be present to not use the default.